androidHyperskillApp/src/main/java/org/hyperskill/app/android/placeholder_new_user/fragment/PlaceholderNewUserFragment.kt

```diff
@@ -1,186 +0,0 @@
-        viewModelFactory = platformPlaceholderNewUserComponent.reduxViewModelFactory
-        viewDataMapper = placeholderNewUserComponent.placeHolderNewUserViewDataMapper
-    }
-
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        setupViewStateDelegate()
-        setupRecyclerView()
-        viewBinding.placeholderError.tryAgain.setOnClickListener {
-            placeholderNewUserViewModel.onNewMessage(
-                PlaceholderNewUserFeature.Message.Initialize(forceUpdate = true)
-            )
-        }
-        placeholderNewUserViewModel.onNewMessage(PlaceholderNewUserFeature.Message.ViewedEventMessage)
-    }
-
-    private fun setupViewStateDelegate() {
-        viewStateDelegate = ViewStateDelegate<PlaceholderNewUserFeature.State>().apply {
-            addState<PlaceholderNewUserFeature.State.Idle>()
-            addState<PlaceholderNewUserFeature.State.Loading>(
-                viewBinding.placeholderContentConstraintLayout,
-                viewBinding.placeholderLoadingView
-            )
-            addState<PlaceholderNewUserFeature.State.NetworkError>(viewBinding.placeholderError.root)
-            addState<PlaceholderNewUserFeature.State.Content>(
-                viewBinding.placeholderContentConstraintLayout,
-                viewBinding.placeholderRecyclerView
-            )
-        }
-    }
-
-    private fun setupRecyclerView() {
-        with(viewBinding.placeholderRecyclerView) {
-            layoutManager = LinearLayoutManager(requireContext(), LinearLayoutManager.VERTICAL, false)
-            adapter = trackAdapter
-
-            val verticalMargin = requireContext().resources.getDimensionPixelSize(R.dimen.track_items_vertical_margin)
-            addItemDecoration(VerticalMarginItemDecoration(verticalMargin = verticalMargin))
-        }
-    }
-
-    override fun onDestroyView() {
-        super.onDestroyView()
-        viewStateDelegate = null
-    }
-
-    override fun onAction(action: PlaceholderNewUserFeature.Action.ViewAction) {
-        when (action) {
-            PlaceholderNewUserFeature.Action.ViewAction.NavigateTo.HomeScreen -> {
-                childFragmentManager
-                    .dismissDialogFragmentIfExists(NewUserTrackDetailsBottomSheet.TAG)
-                requireRouter().newRootScreen(MainScreen)
-            }
-            PlaceholderNewUserFeature.Action.ViewAction.ShowTrackSelectionStatus.Loading -> {
-                LoadingProgressDialogFragment.newInstance()
-                    .showIfNotExists(childFragmentManager, LoadingProgressDialogFragment.TAG)
-            }
-            PlaceholderNewUserFeature.Action.ViewAction.ShowTrackSelectionStatus.Error -> {
-                childFragmentManager
-                    .dismissDialogFragmentIfExists(LoadingProgressDialogFragment.TAG)
-                viewBinding.root.snackbar(org.hyperskill.app.R.string.placeholder_new_user_start_track_error_message)
```

Comment:
Please check maybe we can delete this string resource `placeholder_new_user_start_track_error_message`